src/components/card/card.tsx

```diff
@@ -71,6 +72,8 @@ export class Card implements ConditionalSlotComponent {
    */
   @Prop({ reflect: false }) intlDeselect: string = TEXT.deselect;
 
+  @Prop() thumbnailPosition: LogicalFlowPosition = "block-start";
```

Comment:
Because this prop reorders the DOM for accessibility purposes in addition for visual reasons, it made more sense to me to make it a prop.  We also have other similar types like `Alignment` that mirror this kind of use case as well.  @macandcheese recommended these 4 values for defining the logical position in the document flow for this feature.  Do you think we need to reevaluate this approach?

Reply:
Since terms like `block`, `inline`, `start` and `end` are CSS terms, another way we can make this prop sounds less CSS-like is to reword these terms to be more logical to a user who doesn't understand what these terms mean in CSS.  Suggested rewordings:

- `vertical` in place of `block`
- `horizontal` in place of `inline`
- `before-content` in place of `start`
- `after-content` in place of `end`

Reply:
I'll merge this in for now and we can create a refactor issue to explore this further if you would like.